tests/component-tests/textfieldControl/js/TextfieldStepperTests.js

```diff
@@ -89,7 +89,7 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
 
         fluid.tests.textfieldStepper.verifyState = function (that, expectedValue, incBtnState, decBtnState) {
             jqUnit.assertEquals("The model value should be " + expectedValue, expectedValue, that.model.value);
-            jqUnit.assertEquals("The aria-valuenow should be " + expectedValue, expectedValue.toString(), that.textfield.container.attr("aria-valuenow"));
+            jqUnit.assertEquals("The value should be " + expectedValue, expectedValue.toString(), that.textfield.container.attr("value"));
```

Comment:
@greatislander related to @amb26 comment above about the value of the input field, this should like get the actual value using something like `that.textfield.container.val()`.

Reply:
I've added a test to do this in 06f3533.